docs/sources/github/example-api-responses/original/issue_timeline.json

```diff
@@ -1,4 +1,39 @@
 [
+  {
+    "sha": "66249c83914e7ad8e9976eededda069b936ee426",
+    "node_id": "C_kwDOKU_SqtoAKDY2MjQ5YzgzOTE0ZTdhZDhlOTk3NmVlZGVkZGEwNjliOTM2ZWU0MjY",
+    "url": "https://api.github.com/repos/khnum-heket/test2/git/commits/66249c83914e7ad8e9976eededda069b936ee426",
+    "html_url": "https://github.com/khnum-heket/test2/commit/66249c83914e7ad8e9976eededda069b936ee426",
+    "author": {
+      "name": "AuthorUser",
+      "email": "authorUser@some-domain.com",
```

Comment:
this is just whatever user sets in `.gitconfig` right? not validated or anything, right?

we should duck-type it, and if is formated like an email address, we'll scope it as such, right?

Reply:
Yes, that was due the invalid config rules proxy wasn't doing it. Now with the changes it is what is doing, but I can include a test to cover the case if you want (if work on that way before)